EasePass/Strings/de-DE/Resources.resw

```diff
@@ -925,13 +925,13 @@
     <value>Neues Passwort</value>
   </data>
   <data name="View_SettingsPage_NoPluginsInfo.Text" xml:space="preserve">
-    <value>Keine Plugins installiert, klicke unten zum Herunterladen</value>
+    <value>Keine Plugins installiert, klicken Sie unten zum Herunterladen</value>
   </data>
   <data name="View_SettingsPage_PasswordGenerator.Text" xml:space="preserve">
     <value>Passwort Generator</value>
   </data>
   <data name="View_SettingsPage_PasswordLength.Header" xml:space="preserve">
-    <value>Passwort länge</value>
+    <value>Passwort länge (min: 8)</value>
```

Comment:
Minimum of 8 characters is not needed, because you can change that in settings 😅